lib/index.js

```diff
@@ -13,5 +13,28 @@ module.exports = (dirname, manifestExtras) => {
 
     const manifest = Manifest.concat(manifestExtras || []);
 
-    return Haute(dirname, manifest);
+    return (server, options, next) => {
```

Comment:
Bear in mind that `options` should be an optional argument.  This is only reflected in the docs– I should add a test for it. With promise support, `options` and `next` are now both optional!